bitnami/grafana/values-production.yaml

```diff
@@ -277,3 +277,38 @@ affinity: {}
 ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
 ##
 podAnnotations: {}
+
+## Prometheus metrics
+##
+metrics:
+  enabled: false
+
+  ## Prometheus Operator ServiceMonitor configuration
+  ##
```

Comment:
I think this comment doesn't correspond this block but the "serviceMonitor" one.